### DISCUSSION TITLE
Node process not ending

### DISCUSSION THREAD

**Discussion by joshmedeski** [vercel/turborepo#1820] [Help] [answered]
<img width="852" alt="Screen Shot 2022-08-31 at 1 44 42 PM" src="https://user-images.githubusercontent.com/2036594/187758262-0a11bb15-c8ab-4b18-91b2-fa8c3fe2361a.png">

While working on my yarn workspace monorepo powered by Turbo, I've noticed my machine has these `node` processes are starting up and never ending.

What could be causing this? My setup uses the fish shell inside a tmux instance and my I'm using volta as my node version manager.

This is typically happening when I run turbo build command. It's slowing down my computer because those node processes are eating up the CPU even though I have no active process running in my terminal.

**Comment by anthonyshew** [comment, marked as answer]
If you're still having this issue, please open up a bug report!
